test/extended/router/shard/shard.go

```diff
@@ -32,20 +30,41 @@ var ingressControllerNonDefaultAvailableConditions = []operatorv1.OperatorCondit
 	{Type: "Admitted", Status: operatorv1.ConditionTrue},
 }
 
-func DeployNewRouterShard(oc *exutil.CLI, timeout time.Duration, cfg Config) (string, error) {
-	jsonCfg, err := oc.AsAdmin().Run("process").Args("-f", cfg.FixturePath, "-p",
-		"NAMESPACE=openshift-ingress-operator",
-		"DOMAIN="+cfg.Domain,
-		"TYPE="+cfg.Type).OutputToFile("config.json")
-	if err != nil {
-		return "", err
+func DeployNewRouterShard(oc *exutil.CLI, timeout time.Duration, cfg Config) (*operatorv1.IngressController, error) {
+	ingressCtrl := &operatorv1.IngressController{
```

Comment:
This is actually another dependency on operator.openshift.io which has not been taken into account before. @Miciah any way to replace it with an object which does not depend on any openshift apigroup?